share/completions/udisksctl.fish

```diff
@@ -0,0 +1,67 @@
+set -l cmds help info dump status monitor mount unmount unlock lock loop-setup loop-delete power-off smart-simulate
+
+set __fish_mount_opts async\tUse\ asynchronous\ I/O atime\tUpdate\ time\ on\ each\ access auto\tMounted\ with\ -a defaults\tUse\ default\ options dev\tInterpret\ character/block\ special\ devices exec\tPermit\ executables _netdev\tFilesystem\ uses\ network noatime\tDo\ not\ update\ time\ on\ each\ access noauto\tNot\ mounted\ by\ -a nodev\tDo\ not\ interpret\ character/block\ special\ devices noexec\tDo\ not\ permit\ executables nosuid\tIgnore\ suid\ bits nouser\tOnly\ root\ may\ mount remount\tRemount\ read-only\ filesystem ro\tMount\ read-only rw\tMount\ read-write suid\tAllow\ suid\ bits sync\tUse\ synchronous\ I/O dirsync\tUse\ synchronous\ directory\ operations user\tAny\ user\ may\ mount users\tAny\ user\ may\ mount\ and\ unmount
+
+function __fish_complete_blockdevice
+    set -l cmd (commandline -ct)
+    [ "" = "$cmd" ]
+    and return
+    for f in $cmd*
+        [ -b $f ]
+        and printf "%s\t%s\n" $f "Block device"
+        [ -d $f ]
+        and printf "%s\n" $f/
+    end
+end
+
+function __fish_print_mounted_blockdevice
+    cat /proc/mounts | grep "^/.*" | cut -d " " -f 1
```

Comment:
Basically, yes. Though the "-i" is superfluous and you should `test -r /proc/mounts` before - you don't want errors here (this would also apply to `cat | grep`).

Of course the assumption here is that block devices can't have spaces in their names, which I _believe_ holds, and even if it didn't it would be a very unusual configuration. /etc/mtab replaces spaces with '\040', dunno if it's the same here.